zerver/tornado/event_queue.py

```diff
@@ -1122,6 +1122,9 @@ def process_message_update_event(
     elif "push_notify_user_ids" in event_template:
         online_push_user_ids = set(event_template.pop("push_notify_user_ids"))
 
+    if "online_email_user_ids" in event_template:
+        del event_template["online_email_user_ids"]
+
     stream_name = event_template.get("stream_name")
```

Comment:
This is just so that I could split this commit and the next (without this, the new field would leak to the API and we won't be able to merge this commit independently).

Reply:
Yeah, this makes sense; it could have been cleaner to do a `.pop` and ignore the output, but this is fine.